### ISSUE TITLE
example.py broken

### ISSUE THREAD

**Issue by zbarry** [ericmjl/pyjanitor#39]
```python
import pandas as pd
import janitor as jn

df = pd.read_excel("dirty_data.xlsx")

df = (
    jn.DataFrame(df)
    .clean_names()
    .remove_empty()
    .rename_column("%_allocated", "percent_allocated")
    .rename_column("full_time?", "full_time")
    .coalesce(["certification", "certification.1"], "certification")
    .encode_categorical(["subject", "employee_status", "full_time"])
    .convert_excel_date("hire_date")
)

print(df)
```

```
Traceback (most recent call last):
  File "/home/zachary/projects/pyjanitor/examples/example.py", line 7, in <module>
    jn.DataFrame(df)
AttributeError: module 'janitor' has no attribute 'DataFrame'
```

`DataFrame` is not in the `janitor` namespace. Also, df is already a `DataFrame`, so not sure what the intent would be of doing this, anyway.

Note that if I replace `jn.DataFrame(df)` with simply `df`, I get:

```
Traceback (most recent call last):
  File "/home/zachary/miniconda3/envs/hv/lib/python3.6/site-packages/pandas/core/indexes/base.py", line 3078, in get_loc
    return self._engine.get_loc(key)
  File "pandas/_libs/index.pyx", line 140, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/index.pyx", line 162, in pandas._libs.index.IndexEngine.get_loc
  File "pandas/_libs/hashtable_class_helper.pxi", line 1492, in pandas._libs.hashtable.PyObjectHashTable.get_item
  File "pandas/_libs/hashtable_class_helper.pxi", line 1500, in pandas._libs.hashtable.PyObjectHashTable.get_item
KeyError: 'certification.1'
```

**Comment by ericmjl** [comment]
@zbarry the following line is a vestige of the old API:

```python
df = jn.DataFrame(df)
```

Originally, that subclassed DataFrame was the only way I knew of to get methods attached to the DataFrame.

Once I started using @zsailer's `pandas-flavor`, monkeypatching happens dynamically.

Thanks for the catch, btw!

As for the KeyError, that is a column name missing from the DataFrame. Not sure what's happening there, though. Needs to be debugged.

Gotta run, have a thing in 2 mins. Thanks for the PRs though!

**Comment by ericmjl** [comment]
Fixed by #41.
